lib/datasource/git-tags/index.ts

```diff
@@ -24,16 +23,15 @@ export async function getPkgReleases({
     if (cachedResult) return cachedResult;
 
     // fetch remote tags
-    const lsRemote = await git.listRemote([
-      '--sort=-v:refname',
-      '--tags',
-      lookupName,
-    ]);
+    const lsRemote = await gitRefs.getPkgReleases({ lookupName });
+
     // extract valid tags from git ls-remote which looks like 'commithash\trefs/tags/1.2.3
-    const tags = lsRemote
-      .replace(/^.+?refs\/tags\//gm, '')
-      .split('\n')
+    const refs = lsRemote.releases.map(release => release.gitRef);
+
+    const tags = refs
+      .map(ref => ref.replace(/^tags\//gm, ''))
       .filter(tag => semver.isVersion(tag));
```

Comment:
We should not filter by semver here. It should allow whatever versioning scheme the repo uses.

Reply:
It was there already, I did not add it, shall I remove that?

Reply:
Yes, it might need removing from the git tags datasource this was copy/pasted from too - but not in this PR

Reply:
it is in git-tags only and not in git-refs!

Reply:
@ViralRuparel Yes, but we should remove that in another pr.

Reply:
okay @viceice